services/static-badge/static-badge.tester.js

```diff
@@ -24,7 +24,11 @@ t.create('All one color')
 
 t.create('Not a valid color')
   .get('/badge/label-message-notacolor.json')
-  .expectBadge({ label: 'label', message: 'message', color: null })
+  .expectBadge({
+    label: 'label',
+    message: 'message',
+    color: undefined,
```

Comment:
This one puzzled me a bit. Why's the color value different? 

I'm asking out of curiosity, not suggesting a change or anything

Reply:
Its a good question because I thought this was sensible but looking at it again, I've unwittingly changed what is going on here.

In javascript

```
> typeof null
'object'
```

so previously, this test would have hit the `if (typeof expected === 'object')` case. Now, there is no case for if `expected` is a generic `object`, so `.expectBadge({ label: 'label', message: 'message', color: null })` throws an exception. I changed it, but changing it to `undefined` actually changes the meaning of the test because then we're making _no assertion at all_ about the value of `color` rather than explicitly asserting that it isn't set.

On reflection, the correct way to explicitly assert that `color` is undefined is to use [`.forbidden()`](https://github.com/hapijs/joi/blob/master/API.md#anyforbidden), which I've done in 48a5afc

Reply:
Ah okay, thanks for the explanation!